electron/utils.ts

```diff
@@ -192,11 +194,7 @@ const launchGame = async (appName: string) => {
   const fixedWinePrefix = winePrefix.replace('~', home)
   let wineCommand = `--wine ${wineVersion.bin}`
 
-  let fixedLauncherArgs = launcherArgs
-
-  if (!(await isOnline()) && !fixedLauncherArgs.includes('--offline')) {
-    fixedLauncherArgs = fixedLauncherArgs.trimRight() + ' --offline'
-  }
+  const offlineArg = (!offlineMode && (await isOnline())) ? "" : "--offline";
```

Comment:
I think just for the readability, maybe storing the online status in a variable could be better:
`const online = await isOnline()`